### DISCUSSION TITLE
Is there any way to `auto_insert` with `preselect`?

### DISCUSSION THREAD

**Discussion by zooeywm** [saghen/blink.cmp#584] [Q&A] [answered]
As the documents says, 
https://github.com/Saghen/blink.cmp/blob/12ffc10c6283ac148a89d72b5540d819fc80e2ff/README.md?plain=1#L340

`auto_insert` is cool, and I really want to use it, but without preselect, I can't get the documentation without manually select when I set  `documentation = {auto_show = true}`

**Comment by redoxahmii** [comment]
Have you tried setting :
```lua 
documentation = {
        auto_show = true,
        auto_show_delay_ms = 0,
        update_delay_ms = 0,
      }
```
Because for me using `preselect` it shows the documentation on the first item or what is matched at that time.

**Comment by saghen** [comment, marked as answer]
It's not currently possible but do you mind creating an issue for this? It might be tricky but I'd personally use this as well
